eman2/protocols/protocol_tomo_boxing.py

```diff
@@ -0,0 +1,190 @@
+# **************************************************************************
+# *
+# * Authors:     Adrian Quintana (adrian@eyeseetea.com) [1]
+# *
+# * [1] EyeSeeTea Ltd, London, UK
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+from pyworkflow.utils.properties import Message
+from pyworkflow.gui.dialog import askYesNo
+from pyworkflow.protocol.params import BooleanParam
+from pyworkflow import utils as pwutils
+
+import eman2
+from eman2.convert import loadJson
+
+from tomo.protocols import ProtTomoPicking
+from tomo.objects import Coordinate3D
+
+
+class EmanProtTomoBoxing(ProtTomoPicking):
+    """ Manual picker for Tomo. Uses EMAN2 e2spt_boxer.py.
+    """
+    _label = 'tomo boxer'
+
+    def __init__(self, **kwargs):
+        ProtTomoPicking.__init__(self, **kwargs)
+
+    @classmethod
+    def isDisabled(cls):
+        return not eman2.Plugin.isNewVersion()
```

Comment:
Should we check here the existence of scipion-em-tomo?

Reply:
In theory, this case should never happen because of this check https://github.com/scipion-em/scipion-em-eman2/pull/20/files#diff-7a40dcf1873c3b15575bd4c9dcf7ce7cR38, right?

Reply:
Yes, I think this 'try' is good enough and we can put there all tomography protocols import